src/coding-guidelines/expressions.rst

```diff
@@ -6,6 +6,270 @@
 Expressions
 ===========
 
+
+.. guideline:: Ensure that integer operations do not result in arithmetic overflow
+    :id: gui_dCquvqE1csI3
+    :category: required
+    :status: draft
+    :release: 1.0 - latest
+    :fls: fls_oFIRXBPXu6Zv
+    :decidability: decideable
+    :scope: system
+    :tags: security, performance, numerics
+
+    Eliminate `arithmetic overflow <https://rust-lang.github.io/fls/expressions.html#arithmetic-overflow>`_ of both signed and unsigned integer types. 
+    Arithmetic overflow will panic in debug mode, but wraparound in release mode.
+    Any wraparound behavior must be explicitly specified to ensure the same behavior in both debug and release modes.
+
+    This rule applies to the following primitive types:
+
+    * ``i8``
+    * ``i16``
+    * ``i32``
+    * ``i64``
+    * ``i128``
+    * ``u8``
+    * ``u16``
+    * ``u32``
+    * ``u64``
+    * ``u128``
+    * ``usize``
+    * ``isize``
+
+    .. rationale::
+        :id: rat_LvrS1jTCXEOk
+        :status: draft
+
+        Eliminate arithmetic overflow to avoid runtime panics and unexpected wraparound behavior.
+        Use explicit wrapping or saturation semantics in cases where these behaviors are intentional.
+        Range checking can be used to eliminate the possibility of arithmetic overflow.
```

Comment:
Here I would add the line that I just "suggestion-deleted" from the Amplification. Then adjust the rest of the paragraph to not repeat what you would've just said (since that line would now be here in the Rationale).